install_limelight.sh

```diff
@@ -34,12 +29,10 @@ apt-get autoremove -y
 echo "Installing additional things"
 sudo apt-get update
 apt-get install -y pigpiod pigpio device-tree-compiler libraspberrypi-bin
-apt-get install -y network-manager
-apt-get install -y net-tools
+apt-get install -y network-manager net-tools
 # libcamera-driver stuff
-apt-get install -y libegl1 libopengl0 libopencv-core406 libgl1-mesa-dri libcamera0.1 libgbm1 libatomic1
-# mrcal stuff
-apt-get install -y libcholmod3 liblapack3 libsuitesparseconfig5
+apt-get install -y libegl1 libopengl0 libopencv-core406 libgl1-mesa-dri libcamera0.1 libgbm1
```

Comment:
Out of scope for this, but. I think we can delete the opencv download now.

Reply:
I wondered about that. I'd also like to consolidate `install_pi.sh` and `install_limelight.sh` since they are nearly identical.